src/components/Card.tsx

```diff
@@ -0,0 +1,70 @@
+import { Div, DivProps } from 'honorable'
+import { forwardRef } from 'react'
+
+type CardSize = 'medium' | 'large' | string
+type CardHue = 'default' | 'lighter' | 'lightest' | string
+
+type CardProps = {
+  hue?: CardHue
+  cornerSize?: CardSize
+  clickable?: boolean
+  selected?: boolean
+} & DivProps
+
+const hueToBGColor: { [key in CardHue]: string } = {
+  default: 'fill-one',
+  lighter: 'fill-two',
+  lightest: 'fill-three',
+}
+
+const hueToBorderColor: {
+  [key in CardHue]: string
+} = {
+  default: 'border',
+  lighter: 'border-fill-two',
+  lightest: 'border-input',
+}
+
+const hueToHoverBGColor: {
+  [key in CardHue]: string
+} = {
+  default: 'fill-one-hover',
+  lighter: 'fill-two-hover',
+  lightest: 'fill-three',
```

Comment:
We might need a `fill-three-hover` and `fill-three-selected` since right now the highest hue hover is the same as the default color. There is no hover effect.

Reply:
That's true, but the [design system docs](https://www.notion.so/pluralsh/Plural-Design-System-4a0558fd40794a6aac8045e9baf8cef6#faf65edf756a4dc98df263ebdd9104f8) say "fill-three may not have hover or selected states". I figure we can add that in later if we actually encounter the need for hover states on the lightest state.